### DISCUSSION TITLE
SVT- Play via proxy

### DISCUSSION THREAD

**Discussion by hackintoss** [stefansundin/privatkopiera#94] [Q&A]
Hi 

I have looked at the posts/discussions regarding downloading videos via proxy and it looks like
being a problem. Have also had a look at svt-dl posts on the same issue.
Is it possible?
Any special requirement on the proxy apart from being located in Sweden?

**Comment by stefansundin** [comment]
Hello.

I have written up my tips regarding proxy usage on this page: https://stefansundin.github.io/privatkopiera/proxy.html

It's in Swedish but hopefully you can get the gist by using a translation tool.

Note that some of the websites detect if you're using a VPN or common cloud provider. I don't know if there's any provider that isn't detected (I have only tested a few free options).
